akka-actor-tests/src/test/java/akka/actor/ActorCreationTest.java

```diff
@@ -175,35 +176,32 @@ public void onReceive(Object message) throws Exception {}
 
   @Test
   public void testWrongAnonymousInPlaceCreator() {
-    try {
-      Props.create(
-          Actor.class,
-          new Creator<Actor>() {
-            @Override
-            public Actor create() throws Exception {
-              return null;
-            }
-          });
-      assert false;
-    } catch (IllegalArgumentException e) {
-      assertEquals(
-          "cannot use non-static local Creator to create actors; make it static (e.g. local to a static method) or top-level",
-          e.getMessage());
-    }
+    IllegalArgumentException exception =
```

Comment:
The test failed. I'm little surprised. I tried to save the semantics. Does  it depend on  ActorCreationTest Line 43? I have read [the documentation](https://docs.oracle.com/javase/specs/jls/se8/html/jls-8.html#jls-8.5.1) . It's safe to remove `static` from nested interface. Am I wrong?